step/step.go

```diff
@@ -71,37 +71,44 @@ func (u Updater) Run(config Config) (Result, error) {
 		return Result{}, err
 	}
 
-	var buildVersion int64
-	if config.BuildVersionOffset >= 0 {
-		buildVersion = config.BuildVersion + config.BuildVersionOffset
+	// Check if build version is numeric
+	parsedBuildVersion, err := strconv.ParseInt(config.BuildVersion, 10, 64)
+	if err != nil {
+		u.logger.Infof("Build version is not numeric (%s), skipping version increment.", config.BuildVersion)
```

Comment:
```suggestion
		u.logger.Infof("Provided build version is not numeric (%s), using it as-is without incrementing", config.BuildVersion)
```
Just to make it super obvious what's going to happen